src/conf.c

```diff
@@ -1465,7 +1465,19 @@ int config__read_file_core(struct mosquitto__config *config, bool reload, struct
 								token++;
 							}
 							if(token[0]){
-								config->log_file = mosquitto__strdup(token);
+								const char* home = getenv("HOME");
```

Comment:
There's a path through this code where `config->log_file` doesn't get assigned.
```suggestion
								config->log_file = NULL;
```

Reply:
@MichaelCook
Thank you for your very fast response!
I added two commits to apply your suggestions. I also could squash them into one commit if needed.